e2e-tests/e2e-fixtures/portal-users.fixture.js

```diff
@@ -7,7 +7,7 @@ const USER_WITH_INJECTION = {
   password: 'TestPassword123!',
   firstname: 'test',
   surname: 'injection',
-  bank: 'Bank 1',
+  bank: '9', // Bank 1
```

Comment:
I don't think, this is quite right.
bank should be an object and this user is used for security tests.